docusaurus/docs-internals/editor-controls/data-controls/colorPicker.md

```diff
@@ -0,0 +1,188 @@
+---
+sidebar_position: 5
+toc_max_heading_level: 4
+---
+
+# Color Picker
+
+Brizy's Color Picker control provides a user-friendly color picker field with an integrated opacity slider.
```

Comment:
Brizy's color picker control offers a user-friendly color picker field with an integrated opacity slider, hue slider, and predefined palette colors.